Makefile

```diff
@@ -223,26 +224,29 @@ install.man: docs
 	install ${SELINUXOPT} -m 644 docs/links/*1 -t $(MANDIR)/man1
 
 install.config:
-	install ${SELINUXOPT} -D -m 644 libpod.conf ${SHAREDIR_CONTAINERS}/libpod.conf
-	install ${SELINUXOPT} -D -m 644 seccomp.json $(ETCDIR_LIBPOD)/seccomp.json
-	install ${SELINUXOPT} -D -m 644 crio-umount.conf $(OCIUMOUNTINSTALLDIR)/crio-umount.conf
+	install ${SELINUXOPT} -d -m 755 $(SHAREDIR_CONTAINERS) $(ETCDIR_LIBPOD) $(OCIUMOUNTINSTALLDIR)
+	install ${SELINUXOPT} -m 644 libpod.conf $(SHAREDIR_CONTAINERS)/libpod.conf
+	install ${SELINUXOPT} -m 644 seccomp.json $(ETCDIR_LIBPOD)/seccomp.json
+	install ${SELINUXOPT} -m 644 crio-umount.conf $(OCIUMOUNTINSTALLDIR)/crio-umount.conf
 
 install.completions:
 	install ${SELINUXOPT} -d -m 755 ${BASHINSTALLDIR}
-	install ${SELINUXOPT} -m 644 -D completions/bash/podman ${BASHINSTALLDIR}
+	install ${SELINUXOPT} -m 644 completions/bash/podman ${BASHINSTALLDIR}
 
 install.cni:
-	install ${SELINUXOPT} -D -m 644 cni/87-podman-bridge.conflist ${ETCDIR}/cni/net.d/87-podman-bridge.conflist
+	install ${SELINUXOPT} -d -m 755 ${ETCDIR}/cni/net.d/
+	install ${SELINUXOPT} -m 644 cni/87-podman-bridge.conflist ${ETCDIR}/cni/net.d/87-podman-bridge.conflist
 
 install.docker: docker-docs
-	install ${SELINUXOPT} -D -m 755 docker $(BINDIR)/docker
-	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man1
+	install ${SELINUXOPT} -d -m 755 $(BINDIR) $(MANDIR)/man1
+	install ${SELINUXOPT} -m 755 docker $(BINDIR)/docker
```

Comment:
Any reason for the order swap on these?

Reply:
@mheon Yes, otherwise the new code would fail.  In the new changes, line#241 creates two directories, then line#242 copies docker into the BINDIR. The original code line#238 created the BINDIR and did the docker copy using the -D option which is not supported on all platforms.  Then line#239 created the MANDIR/man1 directory.